pythonScript_ShorterDLYLessEvt.py

```diff
@@ -0,0 +1,732 @@
+                        VCal = (d & 0xff000000) >> 24
+                        if (Eff >= 0.48):
+                            print VCal, " => ",Eff
+                            foundVCal = VCal
+                            break
+                        pass
+                    pass
+                except:
+                    print "Error while reading the data, they will be ignored"
+                    continue
+
+                if (foundVCal > VCal_ref and TRIM_IT < MAX_TRIM_IT and trimDAC < 31):
+                    trimDAC += 1
+                    TRIM_IT +=1
+                elif (foundVCal < VCal_ref and TRIM_IT < MAX_TRIM_IT and trimDAC > 0):
+                    trimDAC -= 1
+                    TRIM_IT +=1
+                else:
+                    g.write(str(trimDAC)+"\n")
+                    TotFoundVCal.append(foundVCal)
+                    f.write("S_CURVE_"+str(channel)+"\n")
+                    for d in data_trim:
+                        f.write(str((d & 0xff000000) >> 24)+"\n")
+                        f.write(str((d & 0xffffff)/N_EVENTS_TRIM)+"\n")
+                        pass
+                    break
+                pass
+            pass
+        m.close()
+        h.write(str(TotFoundVCal)+"\n")
+        h.close()
+        g.close()
+        VCalList = []
+        minVcal = 0
+    ################# Set all the Trim_DAC to the right value #################
+        g=open("%s_TRIM_DAC_value_VFAT_%d_ID_0x%04x"%(str(Date),port,chipIDs[port-VFAT_FIRST]&0xffff),'r')
+        #g=open(str(Date)+"_TRIM_DAC_value_VFAT_"+str(port)+"_ID_"+ str(chipIDs[port]&0xff),'r')
+        for channel in range(CHAN_MIN, CHAN_MAX):
+            if options.debug:
+                if channel > 10:
+                    continue
+                pass
+            regName = "vfat2_" + str(port) + "_channel" + str(channel + 1)
+            trimDAC = (g.readline()).rstrip('\n')
+            print trimDAC
+            regValue = int(trimDAC)
+            glib.set(regName, regValue)
+            #regValue = (1 << 6) + int(trimDAC) # old version
+            pass
+
+        g.close()
+    ########################## Final threshold by VFAT2 ######################
+        f.write("second_threshold\n")
+        glib.set('scan_reset', 1)
+        glib.set('scan_mode', 0)
+        glib.set('scan_vfat2', port)
+        glib.set('scan_min', 0)
+        glib.set('scan_max', 255)
+        glib.set('scan_step', 1)
+        glib.set('scan_n', int(N_EVENTS))
+        glib.set('scan_toggle', 1)
+        while (glib.get("scan_status") != 0): r = 1
+        data = glib.fifoRead('scan_data', 255)
+        for d in data:
+            f.write(str((d & 0xff000000) >> 24)+"\n")
+            f.write(str(100*(d & 0xffffff)/N_EVENTS)+"\n")
+            pass
+        f.close()
+        pass
+    pr.disable()
+    s = StringIO.StringIO()
+    sortby = 'cumulative'
+    ps = pstats.Stats(pr, stream=s).sort_stats(sortby)
+    ps.print_stats()
+    print s.getvalue()
+
```

Comment:
I think this was some old unmerged/uncommitted changes from when we were doing the slice test QC and attempting to troubleshoot.

Most likely I copied the original "pythonScript.py" and did some ill-advised troubleshooting in December and the .gitignore was not set properly and this file was considered.

Will remove and attempt to work on .gitignore a little.